src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -177,7 +177,13 @@ MessageML parse(String message, String entityJson, String version) throws Invali
 
     this.messageML = parseMessageML(expandedMessage, version);
     this.entityJson = this.messageML.asEntityJson(this.entityJson);
+    int entityJsonSize = this.entityJson.toString().length();
 
+    biContext.addItemWithValue("MessageLength", message.length());
+    // if entityJson is empty entityJson.toString={}
+    if(entityJsonSize > 2) {
```

Comment:
Thinking about that while I'm seeing a whitespace missing between `if` and `(` 😄  Could you take the opportunity to add https://github.com/finos/symphony-bdk-java/blob/main/.editorconfig to this repo?

Reply:
yes sure, I'll add the file in the repo! :)